src/query/ts/exemplar.go

```diff
@@ -0,0 +1,10 @@
+package ts
+
+// Exemplar holds the exemplar information for a datapoint.
+type Exemplar []byte
+
+// Exemplars is an array of exemplars.
+type Exemplars []Exemplar
+
+// ExemplarList represents a slice of exemplar list pointers.
+type ExemplarsList []*Exemplars
```

Comment:
Think this should just be `type ExemplarsList []Exemplars` since the slice is a pointer already